src/IBM.Watson.Assistant.v1/AssistantService.cs

```diff
@@ -30,21 +34,22 @@ namespace IBM.Watson.Assistant.v1
 {
     public partial class AssistantService : IBMService, IAssistantService
     {
-        const string serviceName = "assistant";
+        const string defaultServiceName = "assistant";
```

Comment:
There are several services where service name needs to be reverted after generation. assistant, compare comply, nlu and visual recognition. These should fail in integration tests.